bin/src/Flags.purs

```diff
@@ -19,8 +19,8 @@ strict ∷ ArgParser (Maybe Boolean)
 strict =
   ArgParser.flag [ "--strict" ]
     "Promotes project sources' warnings to errors"
-    # ArgParser.boolean
-    # ArgParser.optional
```

Comment:
This always returns `Just false` when the flag is not provided because the combo of `boolean` and then `optional` is:
```
default Nothing $ map Just $ default false $ true <$ flag
-- which simplifies to
map Just $ default false $ true <$ flag
-- which simplifies to
default (Just false) $ Just true <$ flag
```

Thus, the `Just` value would always override the workspace one.